.detoxrc.js

```diff
@@ -0,0 +1,87 @@
+/** @type {Detox.DetoxConfig} */
```

Comment:
Configuration file uses the ios simulator expected in the Detox configuration. Android is different since the Detox documents asked for an android emulator that was not installed in my computer.

Reply:
https://wix.github.io/Detox/docs/introduction/project-setup